back/spec/services/permissions/idea_permissions_service_spec.rb

```diff
@@ -458,7 +485,7 @@
   end
 
   describe 'action_descriptors' do
-    it 'does not run more than 3 queries for 5 ideas in a project with default user permissions' do
+    it 'does not run more than 8 queries for 5 ideas in a project with default user permissions' do
```

Comment:
Wonder why this has gone up so much. Maybe we need to include idea_status in the original query where we did not before?